src/main/java/core/basesyntax/AreaCalculator.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface AreaCalculator {
+    public double getArea();
```

Comment:
declaring access modifiers for methods in interface is redundant, they always `public abstract`
```suggestion
    double getArea();
```